rs/nns/governance/src/neuron_store/metrics.rs

```diff
@@ -98,6 +100,39 @@ impl NeuronMetrics {
         self.public_neuron_subset_metrics
             .increment(voting_power_economics, now_seconds, neuron);
     }
+
+    /// This could modify either declining_voting_power_neuron_subset_metrics, or
+    /// fully_lost_voting_power_neuron_subset_metrics (but not both), since
+    /// those categories are mutually exclusive.
+    fn increment_declining_voting_power_or_fully_lost_voting_power_neuron_subset_metrics(
+        &mut self,
+        voting_power_economics: &VotingPowerEconomics,
+        now_seconds: u64,
+        neuron: &Neuron,
+    ) {
+        let seconds_since_voting_power_refreshed =
+            // Here, we assume that the neuron was not refreshed in the future.
+            // This doesn't always hold in tests though, due to the difficulty
+            // of constructing realistic data/scenarios.
+            now_seconds.saturating_sub(neuron.voting_power_refreshed_timestamp_seconds());
```

Comment:
We have a comment interrupting setting a variable here - could we move the comment up above the let statement?